templates/vue/src/components/tyde/activities/CircleOfSupport/CommunityView/onboarding/Tooltip.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div
+    ref="tooltip"
+    class="tooltip-container"
+    :style="{ opacity: positioned ? 1 : 0 }"
+  >
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+// import Helpers from "@/utils/Helpers"
+export default {
+  props: {
+    /**
+     * ID of the DOM element that we want to position this tooltip against.
+     */
+    // refId: {
+    //   type: String,
+    //   required: false,
+    // },
```

Comment:
If this is not used, it should be removed not commented out. Commented out code should explain why it's commented out if it's something that's needed.